catalog/core/catalog-core-migratable/pom.xml

```diff
@@ -152,7 +152,7 @@
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.70</minimum>
+                                            <minimum>0.65</minimum>
```

Comment:
Test coverage went down. :( I assume due to the refactoring, the ratio of critical path to guff has gone down. Could you confirm that we're not missing anything coverage-wise though?